pkg/config/config.go

```diff
@@ -19,13 +17,13 @@ var MQTTConfigDefaults = MQTTConfig{
 }
 
 var CacheConfigDefaults = CacheConfig{
-	Timeout: 2 + time.Minute,
```

Comment:
Change this please to 2 * time.Minute. The original 2 + time.Minute is wrong. This was  1 minute and 2 microseconds.